packages/react-ui/__tests__/widgets/TableWidgetUI.test.js

```diff
@@ -73,4 +73,17 @@ describe('TableWidgetUI', () => {
       expect(mockOnSetSortDirection).toHaveBeenCalledWith('desc');
     });
   });
+
+  describe('case insensitive', () => {
+    test('should render rows properly', () => {
+      const keptFields = ['address', 'city'];
+      const filteredColumns = columns
+        .filter((col) => keptFields.includes(col.field))
+        .map((c) => ({ ...c, field: c.field.toUpperCase() }));
+      render(<Widget columns={filteredColumns} />);
+
+      const row = rows[1];
+      screen.getByText(row.address);
```

Comment:
This test is lacking an `expect`. I know it will fail if the `getByText` doesn't work, but it's better to be explicit about what we want and to have proper error messages.

Reply:
ups, you're right. Copy-paste

Reply:
fixed